docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  rabbitmq:
+    container_name: rabbitmq
+    image: rabbitmq:3.6.10-management
+    ports:
+    - "34369:4369"
+    - "55672:25672"
+    - "35671:5671"
+    - "35672:5672"
+    - "46671:15671"
+    - "46672:15672"
```

Comment:
Do we need all these ports bound? Might be worth just binding the web GUI one and the main port used by the Python service.

Reply:
fair shout